specs/core/0_beacon-chain.md

```diff
@@ -377,6 +379,16 @@ Unless otherwise indicated, code appearing in `this style` is to be interpreted
 }
 ```
 
+#### `ProofOfPossessionData`
```

Comment:
I would simply reuse `DepositInput `with `proof_of_possession` set to `[0, 0]`.

Reply:
While we're at it, maybe define the constant `EMPTY_SIGNATURE` to be `[0, 0]`. It's used three times.